simuleval/data/dataloader/s2t_dataloader.py

```diff
@@ -58,6 +58,12 @@ def get_video_id(url):
 
 @register_dataloader("speech-to-text")
 class SpeechToTextDataloader(GenericDataloader):
+    def __init__(
+        self, source_list: List[str], target_list: List[str], tgt_lang: str
```

Comment:
We also want things to remain backward compatible, so make the tgt_lang arg optional when it makes sense. 

If some existing pipeline that is using this dataloader without tgt_lang, this change will break those pipelines.

Reply:
Noted

Reply:
If I understand correctly, you mean the tgt_lang here should be optional. right?

Reply:
yes